appliance/test/appliance_config_test.yml

```diff
@@ -14,26 +14,29 @@
 ---
 
 # Password used while deploying OS on bare metal servers and for Cobbler UI.
-# The Length of the password should be more than 7.
+# The Length of the password should be atleast 8.
```

Comment:
Missing a space between "at" and "least"

Reply:
Updated script